config/v1/types_infrastructure.go

```diff
@@ -160,6 +160,9 @@ const (
 
 	// EquinixMetalPlatformType represents Equinix Metal infrastructure.
 	EquinixMetalPlatformType PlatformType = "EquinixMetal"
+
+	// AlibabaCloudPlatformType represents Alibaba Cloud infrastructure
+	AlibabaCloudPlatformType PlatformType = "AlibabaCloud"
```

Comment:
Add a fullstop for consistency.  
```suggestion
	// AlibabaCloudPlatformType represents Alibaba Cloud infrastructure.
	AlibabaCloudPlatformType PlatformType = "AlibabaCloud"
```